HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,706 @@
+6. As an hts-token-airdrop-sender I want to cover the fees (airdrop, custom, association & rent until the end of the first full auto renewal period) of an airdrop that results in a pending transfer to reduce the friction by the intended receiving account.
+7. As an airdrop-sender I want to cancel pending token transfers that I initiated and are still unclaimed by the receiver.
+8. As a token treasury account holder, I want to airdrop tokens with custom fallback fees
+
+#### Receiver
+
+1. As a receiver, I want to be able to query the mirror node to see if I have any pending airdrops targeted for my account.
+2. As a receiver who has pending airdrops, I want to be able to claim the airdrops using a Claim API. I expect to receive this airdrop without paying for association fee and custom fee. I understand that if the sender does not have this token in their account when I claim it or if they have cancelled the pending airdrop, this claim request will fail, and the pending airdrop will be deleted.
+3. As a receiver who claims a token airdrop I will only be responsible for the transaction submission fee. No custom fees will be assessed to me as part of this transfer. 
+4. As a user with receiver signature required enabled any airdrops sent to me will result in a pending transfer, even with open slots.
+
+### General User Stories
+
+1. As an hts-token-holder I want to be able to reject unwanted tokens (regardless of how they were obtained) by sending them back to the treasury account and not having to pay any custom fees.
+2. As a non-hedera-centric wallet user, I want to obtain an auto-created account (by EVM address or public key) that is able to receive unsolicited tokens. I understand this account will have `maxAutomaticAssociations` set to -1 implying unlimited automatic associations.
+3. As account owner for an account created by an EVM tool my experience with airdrops on Hedera is identical to that on Ethereum
+4. As a network user I want to be able to accurately query the mirror node for all pending transactions sent to an account
+5. As a network user I want to be able to accurately query the mirror node for all outstanding airdrops sent by an account
+6. As a wallet user I want my wallet to show me all pending transactions to my account
+7. As an explorer user I want my explorer to show me search or view all pending transactions and outstanding  to my accounts
+8. As a DApp developer I want to expose airdrop HAPI functionality and pending transfers state information to my users
+9. As a HAPI developer I want to be able to query the cost of an airdrop transactions in HBAR
+10. As a DApp developer I want to be able to query the cost of an airdrop transactions in gas
+11. As an account that has open auto association slots, I only pay for the fees and rent for used association slots. I no longer pay for unused association slots.
+
+### FAQ
```

Comment:
Are there semantic or operational _limits_ to anything new in this HIP?  E.g., is there a maximum number of pending transfers a given (sending) account can have?  Is there a maximum over the system (that needs to be specified here)? (My intuition says there are no per-account or per-token-type limits, but I may be wrong.)

Reply:
No account or token limits at this time.
Transfer specification limits will match the CryptoTransfer transaction.
Fees are intended to adequately capture the work and resulting state.